sonarapp/Program.cs

```diff
@@ -20,12 +20,14 @@ static void Main(string[] args)
 
             Boolean verifiyResult = v.verifywhat(string.Empty);
 
+            Console.WriteLine("Comparison result: " + verifiyResult.ToString());
+
         }
     }
 
     public class verify
     {
-        string name;
+        public string name;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this field 'private' and encapsulate it in a 'public' property. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=csharpsquid%3AS1104)